Octokit.Tests.Integration/Reactive/ObservableEventsClientTests.cs

```diff
@@ -617,7 +618,82 @@ public TheGetAllUserPerformedPublicMethod()
 
         public class TheGetAllForAnOrganizationMethod
         {
-           
+            readonly ObservableEventsClient _eventsClient;
+            readonly string _organization;
+            readonly string _user;
+
+            public TheGetAllForAnOrganizationMethod()
+            {
+                var github = Helper.GetAuthenticatedClient();
+                _eventsClient = new ObservableEventsClient(github);
+                _user = github.User.Current().Result.Login;
+                _organization = github.Organization.GetAllForCurrent().Result.First().Login;
+            }
+
```

Comment:
Have you [configured your integration test settings](https://github.com/octokit/octokit.net/blob/master/CONTRIBUTING.md#slow-tests) with a test user (and test organization that this user is an owner of) created specifically for running the integration tests (so as not to mess up your personal account or rate limits etc)?

Does that not work?